src/Persisters/Traits/ResolveValuesHelper.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Persisters\Traits;
+
+use BackedEnum;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Proxy\DefaultProxyClassNameResolver;
+
+use function array_merge;
+use function is_array;
+use function is_object;
+
+trait ResolveValuesHelper
```

Comment:
Please flag the trait as `@internal`!

Reply:
Added

Reply:
@derrabus , I don't know why, but it seems that your comment is applied on the first commit of this PR, so it's not shown as "outdated"